### ISSUE TITLE
[dev.icinga.com #13857] Receiving flapping end alerts, without flapping start events.

### ISSUE THREAD

**Issue by icinga-migration** [Icinga/icinga2#4903]
This issue has been migrated from Redmine: https://dev.icinga.com/issues/13857

**Created by ziesemer on 2017-01-02 20:38:09 +00:00**

Assignee: _(none)_
Status: _New_ 
Target Version: _(none)_
Last Update: _2017-01-04 07:00:26 +00:00 (in Redmine)_

    Icinga Version: 2.6.0
    Backport?: Not yet backported
    Include in Changelog: 1

---

Since upgrading to 2.6.0, we've been receiving surprise FLAPPINGEND notifications, as shown in the below 2 log anonymized log snippets:

    Service Notification[2017-01-02 13:21:31] SERVICE NOTIFICATION: $contact$;$host$;ping;FLAPPINGEND;mail-service-notification;PING OK - Packet loss = 0%, RTA = 6.82 ms - $ip$;
    Service stopped flapping[2017-01-02 13:21:31] SERVICE FLAPPING ALERT: $host$;ping;STOPPED; Checkable appears to have stopped flapping (30% change < 30% threshold)
    Service Ok[2017-01-02 13:15:37] SERVICE ALERT: $host$;ping;OK;HARD;1;PING OK - Packet loss = 0%, RTA = 6.71 ms - $ip$
    Service Warning[2017-01-02 13:15:04] SERVICE ALERT: $host;ping;WARNING;SOFT;1;PING WARNING - Packet loss = 0%, RTA = 153.41 ms - $ip$

    Service Notification[2017-01-01 07:14:13] SERVICE NOTIFICATION: $contact$;$host$;rdp;FLAPPINGEND;mail-service-notification;x224 OK. Connection setup time: 0.005646 sec.;
    Service stopped flapping[2017-01-01 07:14:13] SERVICE FLAPPING ALERT: $host$;rdp;STOPPED; Checkable appears to have stopped flapping (27% change < 30% threshold)
    Service Ok[2017-01-01 07:04:26] SERVICE ALERT: $host$;rdp;OK;HARD;1;x224 OK. Connection setup time: 0.003068 sec.
    Service Warning[2017-01-01 07:03:57] SERVICE ALERT: $host$;rdp;WARNING;SOFT;1;x224 WARNING: RDP connection setup time (4.242577) was longer than (3) seconds

In both cases, there is nothing in the logs for 6+ hours prior - including no FLAPPINGSTART event, which I can't understand why there would be a FLAPPINGEND without a prior FLAPPINGSTART - nor were there any other failed checks that should have caused the checks to be considered as "flapping".

(I can't speak further to as to how this worked in 2.5, as we had to disable flapping detection there due to [\#12560.)](https://dev.icinga.com/issues/12560)

**Comment by dnsmichi** [comment]
Will be continued in #4982.
